docs/source/kiva/compiled_path.rst

```diff
@@ -0,0 +1,25 @@
+CompiledPath
+============
+
+A path is a collection of geometry that can be drawn in a graphics context with
```

Comment:
"collection of geometry" sounds a bit strange to me grammatically.  However, I have never been great with grammar so feel free to ignore

Reply:
Yeah... I was struggling for a concise way to describe a path when I wrote that. Suggestions welcome.

Reply:
Yeah it seems other documentation struggles with describing a path as well:
Quartz docs: "A path defines one or more shapes, or subpaths. A subpath can consist of straight lines, curves, or both. It can be open or closed. A subpath can be a simple shape, such as a line, circle, rectangle, or star, or a more complex shape such as the silhouette of a mountain range or an abstract doodle."
Cairo docs: (taken out of context) "The path is somewhere between part of the mask and part of the context. I will show it as thin green lines on the mask layer. It is manipulated by path verbs, then used by drawing verbs"

I think what you have is fine.  I can't decide if "geometry" -> "geometries" makes more or less sense.

Reply:
Man... those other explanations make me feel a bit better 😅

Maybe: "collection of geometric objects"?

Reply:
I apologize for the late response, but yeah I like geometric objects 👍